packages/grafana-data/src/transformations/transformers/ids.ts

```diff
@@ -30,4 +30,5 @@ export enum DataTransformerID {
   fieldLookup = 'fieldLookup',
   spatial = 'spatial',
   extractFields = 'extractFields',
+  insertValues = 'insertValues',
```

Comment:
Maybe "insertGaps"?  or "insertValueAtInterval"?

Is there a use for this other than adding null?

Reply:
i was thinking maybe 0 or 'previous' (LOCF [1]) could be useful, but maybe not.

insertGaps is rather vis-specific and doesnt really communicate well what is actually happening to the data itself. insertValues (or insertNulls) seemed more clear to me.

https://methods.sagepub.com/reference/encyc-of-research-design/n211.xml

Reply:
removed